config.json

```diff
@@ -57,6 +57,18 @@
           "numbers"
         ]
       },
+      {
+        "slug": "guessing-game",
+        "name": "Guessing Game",
+        "uuid": "fb9868eb-ce4f-4e6b-aa0d-b47d3451070f",
+        "prerequisites": [
+          "numbers"
+        ],
+        "status": "beta",
+        "concepts": [
+          "pattern-matching-literals"
+        ]
+      },
```

Comment:
any standard for ordering concept exercises (specifically within the JSON file only), and are we following that standard?

Reply:
I've not come across a standard for this, though that doesn't mean that there isn't one. I'm willing to take on the work of reordering if we discover that there is a standard.